alephium/contracts/constants.ral

```diff
@@ -24,7 +24,6 @@ Abstract Contract Constants() {
     InvalidTransferTargetChain              = 19
     InvalidDestroyUnexecutedSequenceMessage = 20
     InvalidCaller                           = 21
-    InvalidTransferToAddress                = 22
```

Comment:
May reduce the following error code by 1

Reply:
Done